From 07e28b187d24bbe45670e6b10383e8857050e6c9 Mon Sep 17 00:00:00 2001
From: Jianqun Xu <jay.xu@rock-chips.com>
Date: Thu, 24 Jan 2019 08:37:17 +0800
Subject: [PATCH] drm/rockchip: select DEBUG_FS for ROCKCHIP_DRM_DEBUG

ROCKCHIP_DRM_DEBUG will call API from debugfs, need to select DEBUG_FS.

Change-Id: I06a7c7c2ce9179796e551727fdd2e08313bfe6ff
Signed-off-by: Jianqun Xu <jay.xu@rock-chips.com>
---
 drivers/gpu/drm/rockchip/Kconfig | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/gpu/drm/rockchip/Kconfig b/drivers/gpu/drm/rockchip/Kconfig
index 00350e97f595..3efdad3c8045 100644
--- a/drivers/gpu/drm/rockchip/Kconfig
+++ b/drivers/gpu/drm/rockchip/Kconfig
@@ -18,7 +18,7 @@ config DRM_ROCKCHIP
 
 config ROCKCHIP_DRM_DEBUG
 	tristate "Rockchip DRM debug"
-	depends on DRM_ROCKCHIP
+	depends on DRM_ROCKCHIP && DEBUG_FS
 	help
 	  This option add a debug nod to dump buf from application.
 	  dump path: /data/vop_buf
-- 
2.35.3

